registry/filetypes.json

```diff
@@ -0,0 +1,38 @@
+{
+    "filetypes": [
+        {
+            "name": "Biologic MPR",
```

Comment:
These were just dummy examples intially, but my last couple of commits indeed use these JSON files to populate the registry database. I don't have a real preference over what format we accept (we could just accept both/any compatible) for now, but it remains to be seen where this data should live. It could also be put in a new repo which is the one that actually receives PRs from people to register file types/extractors. At some point we can then decide whether doing everything through GitHub PRs is feasible 🙃

Reply:
I think for now we should be accepting PR's of `filetype` and `extractor` schemas to this repo. If it ever gets hard to handle, that will be a nice problem to have.

Reply:
Alternatively, as these are just examples for now, we could also pull them all from the schemas submodule to keep a single point of truth